addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1975,7 +1975,7 @@ options.registry.HeaderNavbar = options.Class.extend({
         // For all header templates except those of the following selector,
         // change the label of the option to "Mobile Alignment" (instead of
         // "Alignment") because it only impacts the mobile view.
-        if (!this.$target[0].querySelector('#oe_structure_header_default_1, #oe_structure_header_hamburger_1, #oe_structure_header_sidebar_1')) {
+        if (!["'default'", "'hamburger'", "'sidebar'", "'magazine'", "'hamburger-full'"].includes(weUtils.getCSSVariableValue('header-template'))) {
             const alignmentOptionTitleEl = this.el.querySelector('[data-name="header_alignment_opt"] we-title');
             alignmentOptionTitleEl.textContent = _t("Mobile Alignment");
         }
```

Comment:
nitpick:
- Change the comment above (`the following selector` make no sense now)
- `'header-template'` we have to use double quote now
- The line is too long (already the case before but you can take the opportunity to fix :+1: )

Reply:
@Guillaume-gdi thanks for the review, I made the changes.
I didn't provide the steps to replicate as it would be repeating what's already mentioned in the current message.